rasterio/rio/info.py

```diff
@@ -50,13 +55,22 @@
 @click.option('--checksum', 'meta_member', flag_value='checksum',
               help="Print integer checksum of a single band "
                    "(use --bidx).")
+@click.option('--gcps', 'meta_member', flag_value='gcps',
+              help="Print JSON representation of ground control points.")
+@sequence_opt
+@use_rs_opt
+@geojson_type_collection_opt()
+@geojson_type_feature_opt()
+@projection_geographic_opt
+@projection_projected_opt
+@precision_opt
```

Comment:
Since this adds a bunch of new command options, we should probably document in the help string that these 7 options only apply to `--gcps`. 

Ideally we could visually group those commands to indicate this, but I couldn't find a way to make click do that.